proto/hub-genesis/state.proto

```diff
@@ -12,11 +12,20 @@ message State {
   reserved 2;
   // accounts on the Hub to fund with some bootstrapping transfers
   repeated GenesisAccount genesis_accounts = 3 [(gogoproto.nullable) = false];
+  // the number of genesis transfers for which an ack has not yet been received
+  uint64  num_unacked_transfers = 5;
```

Comment:
quite redundent with ` repeated GenesisAccount` 
better to store `unacked_transfer_seq_nums` instead?

Reply:
does https://github.com/dymensionxyz/dymension-rdk/pull/458#discussion_r1650990820 explain?